unity-renderer/Assets/Scripts/MainScripts/DCL/Models/AvatarAssets/EmbeddedEmotesSO.cs

```diff
@@ -16,5 +16,8 @@ public class EmbeddedEmote : WearableItem
     {
         public AnimationClip femaleAnimation;
         public AnimationClip maleAnimation;
+        public bool dontShowInBackpack;
+
+        public override bool DontShowInBackpack() { return dontShowInBackpack; }
```

Comment:
probably a rename improvement could be done here: `ShouldBeVisibleInBackpack` (reverting the condition)

Reply:
Done!